python/grass/pygrass/raster/category.py

```diff
@@ -271,7 +271,8 @@ def copy(self, category):
         :type category: Category object
         """
         libraster.Rast_copy_cats(
-            ctypes.byref(self.c_cats), ctypes.byref(category.c_cats)  # to
```

Comment:
This has wrongly placed comments.

Reply:
Can you take a look at this commit? I think there's a typo since it was introduced 11 years ago : https://github.com/echoix/grass/pull/354/commits/e76cbcb94814acdf881ad9bccdf6d1089d4a46bd

Reply:
> I saw them in [echoix#354](https://github.com/echoix/grass/pull/354) too. They were wrong before.
I'll address after.